docs/content/getting_started/manual_installation.md

```diff
@@ -0,0 +1,311 @@
+# Manual Installation
+{!content/getting_started/minimum_requirements.md!}
+### Installation Instructions
```

Comment:
Generally, you don't need this section heading. The main heading already indicates that this is an installation.